### DISCUSSION TITLE
使用whisper语音转文本，更换模型ggml-base.bin出现错误。

### DISCUSSION THREAD

**Discussion by KunZhong** [ZuodaoTech/everyone-can-use-english#683] [General]
![屏幕截图 2024-06-17 212009](https://github.com/xiaolai/everyone-can-use-english/assets/9493136/e1725f38-fc28-41a2-aa6e-796302d7ea8a)
![屏幕截图 2024-06-17 212229](https://github.com/xiaolai/everyone-can-use-english/assets/9493136/b3c0561a-a094-47da-b07a-a323eb0aa851)

**Comment by an-lee** [comment]
tiny 模型正常吗？可以把模型删了，重新下载再试试。如果还不行，也可能是机器带不动？

**Reply by KunZhong** [reply]
有可能是机器带不动的原因，我用的迷你主机。是最低配置多少才能带动呢？

**Reply by KunZhong** [reply]
我用的是零刻EQ59PRO迷你主机，采用16G+512G存储，处理器是intel11代N5105，运行频率2.0GHz.

**Reply by an-lee** [reply]
那应该就是带不动了。用 tiny 模型一般够了，如果觉得质量不行，可以考虑用 Azure AI
